assets/js/src/context/index.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import getFeaturesContext from './getFeaturesContext.jsx';
+import getConstantsContext from './getConstantsContext.jsx';
+
+/**
+ * Builds the value of the global context.
+ * This starts with `use` because it needs to call
+ * some React hooks to build some parts of the context.
+ */
+export function useGlobalContextValue(data) {
+  const features = getFeaturesContext(data);
+  const constants = getConstantsContext(data);
+  return { features, constants };
```

Comment:
I am thinking about the grouping you did here and I am not sure if it is a good idea or not. It seems to me that we will have a growing `constants` context and not much anything else. I am not sure if that is good or not. Just an idea, not sure if better or worse:
```
context: {
  users: {
    isNewUser: true,
  },
  segments: {
    all: [...],
  },
  features: {},
},
```

Reply:
I am not sure which structure is better for now, because I don't know all constants we put into `window`. My idea was to just create the context now with minimal data. Then after providing it to all components, we can then take a look at all constants we pass into window and define a good structure for the global context. What do you think about that?